docs/crds/avinfrasetting.md

```diff
@@ -148,6 +155,21 @@ AviInfraSetting CRD can be used to enable/disable Public IP on the virtualservic
         network:
           enablePublicIP: true
 
+##### Custom Ports
+
+In order to overwrite the ports opened for VSes created by AKO, users can provide the port details under the `listeners` setting. The ports mentioned under this section overwrites the default open ports, 80 and 443 (SSL enabled). If same port is mentioned in applicable Hostrule then setting mentioned in aviinfrasetting will take precedence. This is applicable only for Shared or Dedicated virtual services.
```

Comment:
should we reword the `applicable only to Shared or Dedicated virtual service` to `L7 virtual service` since the L4 vs also has a concept of sharing the VIP?